user_guide/data_management_view.rst

```diff
@@ -0,0 +1,173 @@
+|
+
+Click the section name and this section will be displayed on the top of the page:
+
+      |
+
+  
+.. image:: ./img/data_management/view/view_page_tabs.png
+
+|
+
+.. _user-guide-ui-components-view-page-actions:
+
+What You Can Do from a View Page
+--------------------------------
+
+Manage the Record from a View Page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Action buttons at the top right of each View page reflect the actions that you can do with the 
+record. 
+
+These include ability to share the record with other users (if applicable), to get to the 
+:ref:`Create/Edit form <user-guide-ui-components-create-pages>`, to delete a record and to perform different 
+:ref:`activities <user-guide-activities>` related to the record (make a task, create a calendar event, log a call, 
+write an email, etc.)
+
+  
+|
+
+.. image:: ./img/data_management/view/view_edit.png
+
+|
+
+
+If there are four or less actions available for the record, a separate button is created for each of them.
+
+|
+
+.. image:: ./img/data_management/view/view_action_buttons_1.png
+
+|
+
+If there are more than four actions available for the record, they will be collected under the 
+*"{Entity Name} Actions*" drop-down.
+
+|
+
+.. image:: ./img/data_management/view/view_action_buttons_2.png
+
+|
+
+Use Contact Details on a View Page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Contact details related to a record are displayed as links. 
+
+You can click the phone number link to call it via Hangouts or log a call, start writing an email with a click on the 
+address, initiate Skype session directly from the View page.
+
+Add a Tag to the Record
+^^^^^^^^^^^^^^^^^^^^^^^
+Tags are non-hierarchical keywords assigned to a record to provide additional information, simplify search and filter 
+records. Tags can be added to a record from its View page, as described in the :ref:`Tags <user-guide-tags-add>` guide.
+
+Share the Details with Other Users
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Sharing records is very convenient when you need assistance from other system users who might have no access to the 
+related record. 
+
+For example, there is a task related to an opportunity that should be performed by a person from a 
+marketing team. Marketing associates don't have access to opportunity records, but the sales manager can share the 
+record with a specific user (or group of users), whose assistance is needed.
+
+To share a record, click the :guilabel:`Share`, and enter the name of the user to share the record with in the 
+*"Share with"* field or click the list icon to select such user(s). 
+
+|
+
+.. image:: ./img/data_management/view/view_share_01.png
+
+|
+
+The selected users will gain access to the record.
+
+Sometimes, there also appears a need to exchange information between organizations. For example, a customer has moved 
+countries, and sales reps of one office want to share the information to the second office.
+Such sharing is possible, but only from a system organization.
```

Comment:
Edit: Sometimes, you may also want to exchange information between organizations. For instance, maybe a customer has moved to a different country, or sales reps at one office need to share some information with a second office. Such sharing is possible, but only from a system organization.